server/workers/countCrimesPerStreet.js

```diff
@@ -1,31 +1,36 @@
 var request = require('request');
 var db = require('../models/rating.js');
 var crime = require('../models/crime');
+var API_KEY = require('../googleMapsConfig.js');
 
 var updateCrimeCounter = function(lat, lng) {
```

Comment:
At some point it may be helpful to separate out this function into smaller sub-functions ... would make it a bit easier to read (can't we re-use the google map api function that lives in utils?)